src/titan_string_bow.cpp

```
@@ -0,0 +1,112 @@
+        }
+    }
+
+    for( item_location &candidate : ammo ) {
+        if( weapon->reload( shooter, candidate, weapon->ammo_required() ) ) {
+            return weapon->ammo_sufficient( &shooter );
+        }
+    }
+    return false;
+}
+
+void spell_effect::titan_string_bow( const spell &sp, Creature &caster, const tripoint &target )
```

Clang (14.0.6)

[error] parameter name 'sp' is too short, expected at least 3 characters

(readability-identifier-length,-warnings-as-errors)